inspire/utils/references.py

```diff
@@ -0,0 +1,77 @@
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+##
```

Comment:
You can remove one of the double `#` in this file to be consistent with the rest.